app/src/main/java/org/gdg/frisbee/android/chapter/InfoFragment.java

```diff
@@ -120,14 +120,14 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
         mInflater = LayoutInflater.from(getActivity());
 
+        final String chapterPlusId = getArguments().getString(Const.EXTRA_PLUS_ID);
```

Comment:
I think with your app indexing commit, this EXTRA_PLUS_ID will be here without a problem. :smile: 
Activity will give it to Fragment.